src/pudl/metadata/templates/codedata.rst.jinja

```diff
@@ -0,0 +1,19 @@
+===============================================================================
+PUDL Code Metadata
+===============================================================================
+
+  *The following tables map from codes used in raw data to labels used in the PUDL database and their descriptions.*
+
+{% for code in CodeData.code_list %}
+
+  | **{{ code.title_name }}**
+  | PUDL Field Name\: {{ code.var_name }}
```

Comment:
I think it might be good to also display at least the fixed codes in the docs -- so people can see how we are re-mapping the originally reported data to the canonical values.